packages/next/package.json

```diff
@@ -72,54 +72,24 @@
   },
   "dependencies": {
     "@babel/runtime": "7.15.4",
-    "@hapi/accept": "5.0.2",
     "@napi-rs/triples": "1.0.3",
     "@next/env": "12.0.8-canary.10",
     "@next/polyfill-module": "12.0.8-canary.10",
     "@next/react-dev-overlay": "12.0.8-canary.10",
     "@next/react-refresh-utils": "12.0.8-canary.10",
-    "acorn": "8.5.0",
-    "assert": "2.0.0",
-    "browserify-zlib": "0.2.0",
     "browserslist": "4.18.1",
-    "buffer": "5.6.0",
     "caniuse-lite": "^1.0.30001283",
     "chalk": "2.4.2",
```

Comment:
In a future PR, we should bump `chalk` since it got much smaller https://packagephobia.com/result?p=chalk

Reply:
Without knowing what we use `chalk` for, what about `picocolors` which claims to be 2 times faster and is ~7kb compared to ~41.

Reply:
`chalk` is used by a bunch of our other dependencies so using the same package de-dupes what is already needed. If `picocolors` has the same API guarantee and we can swap it for sub-dependencies we could look at that